content/about/verslagen/17-18/2018-04-19.md

```diff
@@ -0,0 +1,174 @@
+% Bestuursvergadering 3
+% Isaura Claeys
+% 20/04/2018 - 16:00
+
+# Bestuursvergadering 5
+
+_Datum: 19/04/2018 - 16:00_
+_Status: Gesloten_
+
+**Timetable:**
+
+| Start | Duur | Onderwerp    |
+| ----- | ---- | ------------ |
+| 16:00 | '10  | Intro        |
+| 16:10 | '5   | Financiën    |
+| 16:15 | '5   | Sysadmin     |
+| 16:20 | '5   | Projecten    |
+| 16:25 | '20  | Events       |
+| 16:45 | '15  | Pauze        |
+| 17:00 | '20  | Ereleden     |
+| 17:20 | '20  | Verkiezingen |
+| 17:40 | '20  | Partnerships |
+| 18:00 | ...  | Trivia, ...  |
+
+
+## 0. Verslagen workflow
+Gebruik van hackmd.io: collaboratief verslagen schrijven in markdown. :white_check_mark: 
```

Comment:
Wanneer gebeurd de merge van het vorige verslag?

Reply:
Na goedkeuring op een vergadering.

Reply:
Na goedkeuring op een vergadering is hoe het zou moeten zijn, maar de resultaten van een net gevoerde vergadering horen ook publiek te zijn - gewoon nog niet als 'officieel statement'. Hoe zorgen we dat leden weten dat er een nieuw verslag te goedkeuring staat? Als we dat goed aanpakken is de review-periode ook een communicatieve en overleg-periode naar de leden toe, wat er ook voor zorgt dat het goedkeuren van het vorig verslag dan ook inhoudt de feedback van leden te overlopen, wat ik erg positief vind.

Reply:
Dat is inderdaad een goed idee. Ik hereinner mij juist dat toen @werthen en ik dit aan het bespreken waren dat de optie ook gevallen is om een repository te maken voor verslagen en deze toe te voegen als subrepo aan zeus.ugent.be.

Het loskoppelen van de verslagen heeft meerdere voordelen. O.a. dat we voor ieder nieuw verslag een PR kunnen openen en dat de PR's dus meestal enkel zullen bestaan uit de 'goed te keuren verslagen'. Waardoor geïnteresseerde leden op die manier de net gevoerde vergadering kunnen bekijken.

Eventueel kunnen de todo's voor bestuursleden, vragen vanuit de leden, de agenda voor de volgende vergadering ... als issues op die repository aangemaakt worden. Dit is ongeveer hoe de opleidingscomissie op dit moment werkt en ik vind persoonlijk dat dat nog goed draait.

Reply:
Zouden we dat loskoppelen misschien niet voor alle content doen dan?

De workflow die je voorstelt is dan zo wat de rol van de 'bestuurszaken' repository die vroeger gebruikt werd? Het enige probleem daar lijkt me de fragmentatie van issues tussen verschillende repositories.

Als we dit doen kunnen we g2 gebruiken voor het managen van bestuurstaakjes.

Reply:
Zoals eerder gezegd gaat het loskoppelen van andere content zoals blogposts en events het complexer maken om die content te schrijven (want je moet in meerdere repositories zitten prutsen). Daarnaast is die content niet zo losgekoppeld van de site als verslagen (content heeft styling opties, en verslagen wil je zo uniform mogelijk).